lib/pacemaker/pcmk_rule.c

```diff
@@ -17,25 +17,31 @@
 #include <crm/pengine/rules_internal.h>
 #include <pacemaker-internal.h>
 
+#include "libpacemaker_private.h"
+
 /*!
  * \internal
- * \brief Initialize scheduler data for checking rules
+ * \brief Initialize scheduler data
  *
  * Make our own copies of the CIB XML and date/time object, if they're not
  * \c NULL. This way we don't have to take ownership of the objects passed via
  * the API.
  *
+ * This function is most useful for checking rules and for working with tickets
```

Comment:
I would say it's useful for public API functions that want the caller to retain ownership of a CIB XML argument